modules/global-classes/api.php

```diff
@@ -0,0 +1,204 @@
+<?php
+
+namespace Elementor\Modules\GlobalClasses;
+
+use Elementor\Core\Kits\Documents\Kit;
+use Elementor\Modules\AtomicWidgets\Styles\Style_Schema;
+use Elementor\Modules\AtomicWidgets\Validators\Styles_Validator;
+
+class API {
+	private Repository $repository;
+
+	public function __construct( Kit $kit ) {
+		$this->repository = new Repository( $kit );
+	}
+
+	public function register_hooks() {
+		add_action( 'rest_api_init', fn() => $this->register_routes() );
+	}
+
+	private function all() {
+		try {
+			$all = $this->repository->all();
+		} catch ( \Exception $e ) {
+			return new \WP_Error( 'unexpected_error', 'Getting global classes failed unexpectedly', [ 'status' => 500 ] );
```

Comment:
```suggestion
			return new \WP_Error( 'unexpected_error', __( 'Getting global classes failed unexpectedly', 'elementor' ), [ 'status' => 500 ] );
```

also fo others.

maybe it will be easier to create a const or custom `WP_Error` which represent `Global_Classes_Error`